primerpedia.js

```diff
@@ -62,4 +62,24 @@ function apiRequest(queryString){
 			}
 		}
 	});
-}
\ No newline at end of file
+}
+
+function getQueryVariable(variable) {
+	// http://stackoverflow.com/questions/2090551/parse-query-string-in-javascript
+    var query = window.location.search.substring(1);
+    var vars = query.split('&');
+    for (var i = 0; i < vars.length; i++) {
+        var pair = vars[i].split('=');
+        if (decodeURIComponent(pair[0]) == variable) {
+            return decodeURIComponent(pair[1]);
+        }
+    }
+    return null;
+}
+
+$(document).ready(function() {
+	if (getQueryVariable("q")) {
```

Comment:
I'd rather be explicit and call the parameter "page". It's short enough, and self-explanatory.